playbooks/files/pip-constraints.txt

```diff
@@ -0,0 +1,25 @@
+### Monitorstack plugin framework
```

Comment:
We eventually will be removing most of these when we move to ``monitorstack``. For now, we're maintaining our old existing constraints.